packages/api/src/client/types/com/atproto/repo/batchWrite.ts

```diff
@@ -5,54 +5,49 @@ import { Headers, XRPCError } from '@atproto/xrpc'
 
 export interface QueryParams {}
 
-export interface CallOptions {
-  headers?: Headers;
-  qp?: QueryParams;
-  encoding: 'application/json';
-}
-
 export interface InputSchema {
-  /**
-   * The DID of the repo.
-   */
   did: string;
-  /**
-   * Validate the records?
-   */
   validate?: boolean;
-  writes: (
-    | {
-        action: 'create',
-        collection: string,
-        rkey?: string,
-        value: unknown,
-      }
-    | {
-        action: 'update',
-        collection: string,
-        rkey: string,
-        value: unknown,
-      }
-    | {
-        action: 'delete',
-        collection: string,
-        rkey: string,
-      }
-  )[];
+  writes: (Create | Update | Delete)[];
+  [k: string]: unknown;
 }
 
-export interface OutputSchema {
-  [k: string]: unknown;
+export interface CallOptions {
+  headers?: Headers;
+  qp?: QueryParams;
+  encoding: 'application/json';
 }
 
 export interface Response {
   success: boolean;
   headers: Headers;
-  data: OutputSchema;
 }
 
 export function toKnownErr(e: any) {
   if (e instanceof XRPCError) {
   }
   return e
 }
+
+export interface Create {
+  action: string;
```

Comment:
I think we want this to reflect the const listed in the schema `action: 'create'` .

Reply:
Fixed